Description: Use UglifyJS (not closure-compiler)
 The Javascript transformation of closure-compiler is not in Debian,
 and the underlying Java implementation is extremely old.
 .
 Also, the claimed "half the size that Uglify.js did" (in release notes
 for v0.12.0 - since removed) evidently is not true.
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
Last-Update: 2017-04-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cakefile
+++ b/Cakefile
@@ -146,12 +146,7 @@
     }(this));
   """
   unless process.env.MINIFY is 'false' or process.versions.node.split('.')[0] is '0'
-    # Google Closure Compiler needs Node >= 4
-    {compiledCode: code} = require('google-closure-compiler-js').compile
-      jsCode: [
-        src: code
-        languageOut: if majorVersion is 1 then 'ES5' else 'ES6'
-      ]
+    {code} = require('uglify-js').minify code, fromString: true
   outputFolder = "docs/v#{majorVersion}/browser-compiler"
   fs.mkdirSync outputFolder unless fs.existsSync outputFolder
   fs.writeFileSync "#{outputFolder}/coffee-script.js", header + '\n' + code
